server/embed/config.go

```diff
@@ -347,22 +347,22 @@ type Config struct {
 	ListenMetricsUrls     []url.URL
 	ListenMetricsUrlsJSON string `json:"listen-metrics-urls"`
 
-	// ExperimentalEnableDistributedTracing indicates if experimental tracing using OpenTelemetry is enabled.
-	ExperimentalEnableDistributedTracing bool `json:"experimental-enable-distributed-tracing"`
-	// ExperimentalDistributedTracingAddress is the address of the OpenTelemetry Collector.
-	// Can only be set if ExperimentalEnableDistributedTracing is true.
-	ExperimentalDistributedTracingAddress string `json:"experimental-distributed-tracing-address"`
-	// ExperimentalDistributedTracingServiceName is the name of the service.
-	// Can only be used if ExperimentalEnableDistributedTracing is true.
-	ExperimentalDistributedTracingServiceName string `json:"experimental-distributed-tracing-service-name"`
-	// ExperimentalDistributedTracingServiceInstanceID is the ID key of the service.
+	// EnableDistributedTracing indicates if tracing using OpenTelemetry is enabled.
```

Comment:
Let's have a separate PR to stabilize distributing tracing feature. We still have some work before stabilization.

Reply:
I get it. So this PR should just work on updating the version of OpenTelemetry right?
If so, I'll cancel the changes of all stabilization work ~

Reply:
Yes, we can update OpenTelemetry now and decide separably to stabilize the tracing.